frontend/www/js/omegaup/arena/arena.js

```diff
@@ -1314,6 +1373,24 @@ export class Arena {
       self.updateClarification(data.clarifications[i]);
     }
 
+    if (self.commonNavbar !== null) {
+      // Removing to the notifications list all unsolved clarifications
+      self.commonNavbar.initialClarifications = data.clarifications
+        .filter(clarification =>
+          self.problemsetAdmin
+            ? clarification.answer === null
+            : clarification.answer !== null,
+        )
+        .reverse();
+
+      // Removing all unanswered clarifications from the notifications list.
+      self.commonNavbar.initialClarifications = self.commonNavbar.initialClarifications.filter(
+        clarification =>
+          localStorage.getItem(
+            `clarification-${clarification.clarification_id}`,
+          ) === null,
+      );
```

Comment:
algo que se me olvidó preguntar antes, pero será posible hacer esto en una sóla pasada?

```suggestion

      self.commonNavbar.initialClarifications = data.clarifications
        .filter(clarification =>
          // Removing all unsolved clarifications.
          self.problemsetAdmin
            ? clarification.answer === null
            : clarification.answer !== null &&
          // Removing all unanswered clarifications.
          localStorage.getItem(
            `clarification-${clarification.clarification_id}`,
          ) === null,
        )
        .reverse();
```

Reply:
Listo